images/rhel/rhel8/ubi8-toolbox/Containerfile

```diff
@@ -0,0 +1,28 @@
+FROM registry.access.redhat.com/ubi8/ubi
+
+ENV NAME=ubi8-toolbox VERSION=8
+LABEL com.github.containers.toolbox="true" \
+      com.github.debarshiray.toolbox="true" \
```

Comment:
We should no longer use the `com.github.debarshiray.toolbox` label.

It's only there for the `fedora-toolbox` images for historical purposes because it matched Toolbox's namespace before it got moved into the Containers organization on GitHub. We should move everything over to the new `com.github.containers.toolbox` label.

Reply:
Removed the label.